pages.zh/common/rsync.md

```diff
@@ -0,0 +1,37 @@
+# rsync
+
+> 一种快速，通用，远程（和本地）文件复制工具
+> 更多信息：<https://linux.die.net/man/1/rsync>
```

Comment:
1. Per #5143 we're supposed to try to move away from die.net :)
2. Missing period.
```suggestion
> 更多信息：<http://manpages.ubuntu.com/manpages/bionic/man1/rsync.1.html>。
```

Reply:
man7 is better because is distro agnostic:
```suggestion
> 更多信息：<https://man7.org/linux/man-pages/man1/rsync.1.html>
```

Reply:
I'll remember that next time!

Reply:
I'll also edit your comment @navarroaxel to add the Chinese period.

Reply:
Thanks, @bl-ue and @navarroaxel . It has been modified